### PR TITLE
Vultr: Introducing vultr_block_storage module

### PR DESCRIPTION
**Pull request by Spredzy**
##### SUMMARY

This commit introduces a new module called vultr_block_storage

It allows a user to manage block storage volumes on the Vultr cloud.

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

  - vultr_block_storage

##### ANSIBLE VERSION

  - devel

##### ADDITIONAL INFORMATION

  - N/A

### CODE REVIEW

**Comment by resmo** [comment]
closes #44232

File: lib/ansible/modules/cloud/vultr/vultr_block_storage.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+class AnsibleVultrBlockStorage(Vultr):
+
+    def __init__(self, module):
+        super(AnsibleVultrBlockStorage, self).__init__(module, "vultr_block_storage")
+
+        self.returns = {
+            'SUBID': dict(key='id'),
+            'label': dict(key='name'),
+            'DCID': dict(key='region', transform=self._get_region_name),
+            'attached_to_SUBID': dict(),
```
Comment (resmo) [inline lib/ansible/modules/cloud/vultr/vultr_block_storage.py]: following our naming, this would be `attached_to_id`. Any concerns changing this?

Comment (Spredzy) [reply lib/ansible/modules/cloud/vultr/vultr_block_storage.py]: done.

File: lib/ansible/modules/cloud/vultr/vultr_block_storage.py [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+        super(AnsibleVultrBlockStorage, self).__init__(module, "vultr_block_storage")
+
+        self.returns = {
+            'SUBID': dict(key='id'),
+            'label': dict(key='name'),
+            'DCID': dict(key='region', transform=self._get_region_name),
+            'attached_to_SUBID': dict(key='attached_to_id'),
+            'cost_per_month': dict(convert_to='float'),
+            'date_created': dict(),
+            'id': dict(),
```
Comment (resmo) [inline lib/ansible/modules/cloud/vultr/vultr_block_storage.py]: is this by mistake? 2 times key `id`: subid and id?

Comment (Spredzy) [reply lib/ansible/modules/cloud/vultr/vultr_block_storage.py]: Yep, mistake. Fixed.

**Review by resmo — APPROVED** [review]
shipit

**Pull request merged**